enterprise/app/jobs/monitor_sla_job.rb

```diff
@@ -0,0 +1,84 @@
+class MonitorSlaJob < ApplicationJob
+  queue_as :scheduled_jobs
+
+  def perform
+    AppliedSla.includes(:conversation, :sla_policy).where(sla_status: 'active').find_each(batch_size: 100) do |applied_sla|
```

Comment:
account.applied_slas 

----

split this into two jobs where main jobs schedules this for each account.